monkey/tests/unit_tests/infection_monkey/master/test_ip_scanner.py

```diff
@@ -222,33 +222,6 @@ def _callback(*_):
     assert stoppable_callback.call_count == 2
 
 
-def test_interrupt_port_scanning(callback, scan_config, stop):
-    def stoppable_scan_tcp_port(port, *_):
-        # Block all threads here until 2 threads reach this barrier, then set stop
-        # and test that neither thread scans any more ports
-        stoppable_scan_tcp_port.barrier.wait()
-        stop.set()
-
-        return PortScanData(port, False, None, None)
-
-    stoppable_scan_tcp_port.barrier = Barrier(2)
-
-    puppet = MockPuppet()
-    puppet.scan_tcp_port = MagicMock(side_effect=stoppable_scan_tcp_port)
-
-    addresses = [
-        NetworkAddress("10.0.0.1", None),
-        NetworkAddress("10.0.0.2", None),
-        NetworkAddress("10.0.0.3", None),
-        NetworkAddress("10.0.0.4", None),
-    ]
-
-    ns = IPScanner(puppet, num_workers=2)
-    ns.scan(addresses, scan_config, callback, stop)
-
-    assert puppet.scan_tcp_port.call_count == 2
-
-
```

Comment:
We should just change this test to assert that no fingerprinters were called. We can adjust the name and any relevant comments.